src/components/call/CallLaneTabs.scss

```diff
@@ -0,0 +1,32 @@
+.CallLaneTabs {
```

Comment:
Tab width in this file is not consistent with other files. Normally we use 4 spaces, and you seem to have done so in most places, but here it seems to be just two.